Types prevent trapFocus={false} in CookieBanner

### Severity

🟡 Medium: Partial functionality broken

### Version

1.3.0

### Steps to Reproduce

1. write `<CookieBanner trapFocus={false} />`
2. run tsc

### Expected Behavior

It should be possible to write `trapFocus={false}` without a typing error.

### Actual Behavior

```
src/main.tsx:107:15 - error TS2322: Type 'false' is not assignable to type 'true'.

107               trapFocus={false}
                  ~~~~~~~~~
```

### Additional Context

_No response_

--- a/packages/react/src/components/cookie-banner/cookie-banner.tsx
+++ b/packages/react/src/components/cookie-banner/cookie-banner.tsx
@@ -95,7 +95,7 @@ export interface CookieBannerProps {
 	 * @remarks Useful for implementing a cookie banner that traps focus
 	 * @default true
 	 */
-	trapFocus?: true;
+	trapFocus?: boolean;
 }
 
 /**

